src/remote-reply/remote-reply.js

```diff
@@ -0,0 +1,117 @@
+	const errorText = __( 'Error', 'activitypub' );
+	const invalidText = __( 'Invalid', 'activitypub' );
+	const [ buttonText, setButtonText ] = useState( replyText );
+	const [ buttonIcon, setButtonIcon ] = useState( copy );
+	const ref = useCopyToClipboard( selectedComment, () => {
+		setButtonIcon( check );
+		setTimeout( () => setButtonIcon( copy ), 1000 );
+	} );
+
+	const [ remoteProfile, setRemoteProfile ] = useState( '' );
+	const retrieveAndFollow = useCallback( () => {
+		let timeout;
+		if ( ! ( isUrl( remoteProfile ) || isHandle( remoteProfile ) ) ) {
+			setButtonText( invalidText );
+			timeout = setTimeout( () => setButtonText( replyText ), 2000 );
+			return () => clearTimeout( timeout );
+		}
+		const path = `/${ namespace }/comments/${commentId}/remote-reply?resource=${ remoteProfile }`;
+		setButtonText( loadingText );
+		apiFetch( { path } ).then( ( { url } ) => {
+			setButtonText( openingText );
+			setTimeout( () => {
+				window.open( url, '_blank' );
+				setButtonText( replyText );
+			}, 200 );
+		} ).catch( () => {
+			setButtonText( errorText );
+			setTimeout( () => setButtonText( replyText ), 2000 );
+		} );
+	}, [ remoteProfile ] );
+
+	return (
+		<div className="activitypub__dialog">
+			<div className="activitypub-dialog__section">
+				<h4>{ __( 'The Comment-URL', 'activitypub' ) }</h4>
+				<div className="activitypub-dialog__description">
+					{ __( 'Copy and paste the Comment-URL into the search field of your favorite fediverse app or server to reply to this Comment.', 'activitypub' ) }
+				</div>
+				<div className="activitypub-dialog__button-group">
+					<input type="text" value={ selectedComment } readOnly />
+					<Button ref={ ref }>
+						<Icon icon={ buttonIcon } />
+						{ __( 'Copy', 'activitypub' ) }
+					</Button>
+				</div>
+			</div>
+			<div className="activitypub-dialog__section">
+				<h4>{ __( 'Your Profile', 'activitypub' ) }</h4>
+				<div className="activitypub-dialog__description">
+					{ createInterpolateElement(
+						__( 'Or, if you know your own profile, we can start things that way! (eg <code>https://example.com/yourusername</code> or <code>yourusername@example.com</code>)', 'activitypub' ),
+						{ code: <code /> }
+					) }
+				</div>
+				<div className="activitypub-dialog__button-group">
+					<input
+						type="text"
+						value={ remoteProfile }
+						onKeyDown={ ( event ) => { event?.code === 'Enter' && retrieveAndFollow() } }
+						onChange={ e => setRemoteProfile( e.target.value ) }
+					/>
+					<Button onClick={ retrieveAndFollow }>{ buttonText }</Button>
+				</div>
+			</div>
+		</div>
+	);
+}
+
+export default function RemoteReply( { selectedComment, commentId } ) {
+	const [ isOpen, setIsOpen ] = useState( false );
+	const title = __( 'Remote Reply', 'activitypub' );
+
+	return(
+		<>
+			<a href="javascript:;" className="comment-reply-link activitypub-remote-reply__button" onClick={ () => setIsOpen( true ) } >
```

Comment:
My devtools were complaining about this URL. But you still want your mouse cursor to behave as expected for a link, despite it not taking you anywhere. A few thoughts:

1. Firstly, we could just use `selectedComment` as the href? Graceful degradation?
2. If you don't want a link at all, let's try with the `Button` component:

```js
 // You're already importing `Button` from `@wordpress/components` above so no need to add it
<>
			<Button isLink className="comment-reply-link activitypub-remote-reply__button" onClick={ () => setIsOpen( true ) } >
			{ is Open && ( // ...
```